docs/quick-start/installation.mdx

```diff
@@ -109,8 +109,8 @@ and set a global version:
 ```bash
 asdf global starknet-foundry 0.27.0
 ```
-</details>
 
+</details>
 
 ## Install RPC
```

Comment:
This comment is out of the scope of the PR. I suggest to change the `Install RPC` with `Verify RPC specific version` and I think ,most of the rpc providers moved into v `0.7.1`. We can open another issue for this. Since we have to fix it on `scaffold-starl` repo as well